android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -244,12 +248,31 @@ public void handleMessage(Message msg) {
                         }
                         msg = obtainMessage(SHOW_PROGRESS);
                         sendMessageDelayed(msg, Math.round(mProgressUpdateInterval));
+                        setFrame();
```

Comment:
Is it possible to decorrelate frame management & time position management. ?
If use set 10 msec for progress update because he wants to be accurate, I don't think he needs a new frame every 10ms.
You just need to accept an object in PROP_FRAME_QUALITY to add quality & frequency